flagsmith-jira-app/src/frontend/components/IssueFeatureTables.tsx

```diff
@@ -147,31 +145,60 @@ const IssueFeatureTable = ({
   // get id and name from first feature - assume non-empty
   const featureId = environmentFeatures[0]!.id;
   const featureName = environmentFeatures[0]!.name;
+  const featureProjectId = environmentFeatures[0]!.project;
+  const projectUrl = `${FLAGSMITH_APP}/project/${featureProjectId}`;
 
   /** Read feature state for each environment */
-  const readFeatureState = useCallback(
-    async (): Promise<FeatureState> => ({
-      featureId,
-      environments: await Promise.all(
-        environments.map(async (environment) => ({
-          ...(await readEnvironmentFeatureState({
+  const readFeatureState = useCallback(async (): Promise<FeatureState> => {
+    const matchingEnvs = environments.filter(
+      (env) => String(env.project) === String(featureProjectId),
+    );
+
+    const environmentFeatureStates = await Promise.all(
+      matchingEnvs.map(async (environment) => {
+        try {
+          const state = await readEnvironmentFeatureState({
             envApiKey: String(environment.api_key),
             featureName,
-          })),
-          name: environment.name,
-          api_key: String(environment.api_key),
-        })),
-      ),
-      counts: environmentFeatures.map((feature) => ({
+          });
+
+          return {
+            ...state,
+            name: environment.name,
+            api_key: String(environment.api_key),
+          };
+        } catch (err) {
+          return null;
```

Comment:
Minor: What would be the possible errors we are expecting here? Can we log them maybe to ease debugging?